snackbars/src/main/java/com/decathlon/vitamin/compose/snackbars/VitaminSnackbars.kt

```diff
@@ -0,0 +1,72 @@
+package com.decathlon.vitamin.compose.snackbars
+
+import androidx.compose.material.Snackbar
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import com.decathlon.vitamin.compose.foundation.VitaminTheme
+
+object VitaminSnackbars {
+    @Composable
+    fun Primary(
+        text: String,
+        modifier: Modifier = Modifier,
+        action: @Composable (VitaminSnackbarsActions.() -> Unit)? = null,
```

Comment:
Please follow the parameter ordering described in [CONVENTIONS.md file](https://github.com/Decathlon/vitamin-compose/blob/main/CONVENTIONS.md#parameter-ordering)

* actionOnNewLine
* colors
* action